metrics/web/lib/base/presentation/graphs/circle_percentage.dart

```diff
@@ -37,28 +37,22 @@ class CirclePercentage extends StatefulWidget {
 
   /// Creates the circle percentage graph.
   ///
-  /// The [value] must be either `null` or from 0.0 (inclusive) to 1.0 (inclusive).
-  /// The [placeholder] must not be `null`. The [placeholder] is displayed,
-  /// if the [value] is `null`.
-  ///
```

Comment:
It feels like that we need this description still (or at least some indication how it works - that it fits the value in some bounds, etc..)